content/root/platform-customization.textile

```diff
@@ -0,0 +1,147 @@
+
+To enable a custom environment for your enterprise, speak to your customer success manager. They will create the custom environment for you and provide you with a @ClientOptions@ object, containing the following properties:
+
+- @environment@: the name of your custom environment.
+- @fallbackHostsUseDefault@ (optional): whether to use the standard fallback hosts when an error requires the use of an alternative host.
+- @fallbackHosts@ (optional): an array of custom fallback hosts to try if an error requires the use of an alternative host.
+
+h3. 3. Test the custom environment
+
+Test your custom environment by visiting your new endpoints. The URL for your endpoints will depend on whether you have opted to use an Ably subdomain or a custom CNAME domain.
+
+For example, if you have elected to use an Ably subdomain, you should test your environment by replacing @[ENVIRONMENT]@ with the value of the @environment@ property in the @ClientOptions@ object:
+
+- *Realtime*: @https://[ENVIRONMENT]-realtime.ably.io/time@ (e.g. https://environment-example-realtime.ably.io/time)
+- *REST*: @https://[ENVIRONMENT]-rest.ably.io/time@ (e.g. https://environment-example-rest.ably.io/time)
+
+Repeat this step for all fallback hosts in the @fallbackHosts@ array, if provided.
+
+h3. 4. Modify your code to use your custom environment
+
+Now that you have tested your custom endpoints, you must modify your code to instantiate the Ably client with the provided @ClientOptions@.
+
+The following demonstrates how you might do this using the JavaScript SDK and an Ably subdomain. Even though the sample code is shown in JavaScript, the same options are common to all the client library SDKs:
+
+```[javascript]
+const ably = new Ably.Realtime({
+  authUrl: '/auth',  <-- your chosen authentication method
+  environment: 'environment-example',
+  fallbackHosts: [
+    'environment-example-a-fallback.ably-realtime.com',     'environment-example-b-fallback.ably-realtime.com'
+  ]
+});
+```
+
+You should then test that all client SDKs can connect, publish, and receive messages (where applicable) before rolling out these changes to production code. You can see an example test (again, using JavaScript) "here":https://jsbin.ably.io/etadus/2/edit.
+
+Where possible, you should also inspect your network traffic to verify that the client SDKs are calling the endpoints for your custom environment. Instead of connections to @realtime.ably.io@ you should see connections to @[ENVIRONMENT]-realtime.ably.io@, or your own custom CNAME domain.
+
+h3. 5. Roll out your changes
+
+Once all your client library SDKs are using the new environment and traffic is arriving at the correct endpoints, the team at Ably will be able to actively reroute your traffic based on your environment settings.
+
+If you have regional constraints on your account, then please contact your customer success manager who will ensure that the regional constraints are applied and tested too.
```

Comment:
Might also be worth mentioning how someone can reach out if they aren't an enterprise customer and are interested (i.e. speaking with the sales team generally or assigned AE if a business customer)

Reply:
Added a section in the overview that reads: "If you are not currently an enterprise customer but are interested in finding out more, then please "get in touch":https://ably.com/contact."